edi_bank_statement_import/data/data.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
```

Comment:
Do you really think we need a specific backend type for this?
We should probably have a generic one into a new module `edi_bank`?
In any case, if you keep it, please move it to its own file and make it noupdate.